src/upstart/ceph-osd.conf.in

```diff
@@ -15,7 +15,7 @@ pre-start script
 
     install -d -m0770 -o ceph -g ceph /var/run/ceph
 
-    /usr/lib/ceph/ceph-osd-prestart.sh --cluster="${cluster:-ceph}" -i "$id"
+    @libexecdir@/ceph/ceph-osd-prestart.sh --cluster="${cluster:-ceph}" -i "$id"
```

Comment:
could you use `CMAKE_INSTALL_FULL_LIBEXECDIR` instead? the reason we have `libexecdir` is for the ease of transition from autotools to cmake. because some of our .in files are still using the automake variables. we should start using the cmake variables in new .in files.